packages/salesforcedx-vscode-core/src/commands/forceStartApexDebugLogging.ts

```diff
@@ -115,7 +116,25 @@ export class ForceStartApexDebugLoggingExecutor extends SfdxCommandletExecutor<{
 
 export async function getUserId(projectPath: string): Promise<string> {
   const defaultUsernameOrAlias = await getDefaultUsernameOrAlias();
-  const username = await OrgAuthInfo.getUsername(defaultUsernameOrAlias!);
+  if (isNullOrUndefined(defaultUsernameOrAlias)) {
+    const err = nls.localize('error_no_default_username');
+    telemetryService.sendErrorEvent(
+      'Undefined username or alias when starting Apex Replay Debugger',
+      err
+    );
+    throw new Error(err);
+  }
+
+  const username = await OrgAuthInfo.getUsername(defaultUsernameOrAlias);
```

Comment:
Is this being called to guarantee we get a username back and not an alias?

Reply:
Yeah, the getUsername method returns the associated username if it's provided an alias. Otherwise, it'll return the same username back